docs/DeveloperGuide.md

```diff
@@ -1,38 +1,312 @@
 # Developer Guide
 
+* [Acknowledgements](#acknowledgements)
+* [Setting up, getting started](#setting-up-getting-started)
+* [Design](#design)
+    * [Architecture](#architecture)
+    * [UI Component](#ui-component)
+    * [Parser Component](#parser-component)
+    * [Recipe Component](#recipe-component)
+    * [RecipeList Component](#recipelist-component)
+    * [WeeklyPlan Component](#weeklyplan-component)
+    * [Database Component](#database-component)
+* [Implementation](#implementation)
+    * [Add Recipes Feature](#add-recipes-feature)
+    * [Edit Recipes Feature](#edit-recipes-feature)
+    * [Categorise/Tag Recipes Feature](#categorisetag-recipes-feature)
+    * [List Recipes Feature](#list-recipes-feature)
+    * [Delete Recipes Feature](#delete-recipes-feature)
+
+* [Appendix: Requirements](#appendix-requirements)
+    * [Product scope](#product-scope)
+    * [Target user profile](#target-user-profile)
+    * [Value proposition](#value-proposition)
+    * [User Stories](#user-stories)
+    * [Non-Functional Requirements](#non-functional-requirements)
+    * [Glossary](#glossary)
+
+---
+
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries --
+include links to the original source as well}
+
+---
+
+## Setting up, getting started
+
+---
```

Comment:
As a developer, I would not know how to set up. Perhaps it is good to fill up this set up guide of the developer guide.